Acme/Certificate/CertificateMetadata.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/*
+ * This file is part of the ACME PHP library.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AcmePhp\Bundle\Acme\Certificate;
+
+use AcmePhp\Core\Ssl\Certificate;
+
+/**
+ * Entity representing a certificate.
+ *
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+class CertificateMetadata
```

Comment:
I was talking about setters, but I thought the same: a single parser is enough IMO, so the merge function could be removed.